### ISSUE TITLE
There is no further progress in the code

### ISSUE THREAD

**Issue by HyeokJaeKiM** [Nanne/pytorch-NetVlad#89]
python main.py --mode=train --arch=vgg16 --pooling=netvlad --num_clusters=64

I tried above codes but theis no answer and just keep loading below ..

(delg_env) C:\Users\User>python main.py --mode=train --arch=vgg16 --pooling=netvlad --num_clusters=64 --nGPU 1
Namespace(arch='vgg16', batchSize=4, cacheBatchSize=24, cachePath='C:\\Temp', cacheRefreshRate=1000, ckpt='latest', dataPath='/nfs/ibrahimi/data/', dataset='pittsburgh', evalEvery=1, fromscratch=False, lr=0.0001, lrGamma=0.5, lrStep=5, margin=0.1, mode='train', momentum=0.9, nEpochs=30, nGPU=1, nocuda=False, num_clusters=64, optim='SGD', patience=10, pooling='netvlad', resume='', runsPath='/nfs/ibrahimi/runs/', savePath='checkpoints', seed=123, split='val', start_epoch=0, threads=8, vladv2=False, weightDecay=0.001)
===> Loading dataset(s)
====> Training query set: 7320
===> Evaluating on val set, query count: 7608
===> Building model
===> Training model
===> Saving state to: /nfs/ibrahimi/runs/Sep05_11-39-33_vgg16_netvlad
C:\Users\User\anaconda3\envs\delg_env\lib\site-packages\torch\optim\lr_scheduler.py:129: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "
C:\Users\User\anaconda3\envs\delg_env\lib\site-packages\torch\optim\lr_scheduler.py:154: UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.
  warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)
====> Building Cache

**Comment by Nanne** [comment]
Are you sure those paths (`dataPath` and `runsPath`) are correct for your machine?

Besides that, the building cache phase does take some time, and there isn't any intermediate output whilst that is ongoing. You could try monitoring CPU/GPU activity to see if the process is still active, additionally you can look in the `cachePath` folder to see if the .hdf5 file is created.

You could also add something like 
```
if iteration % 10 == 0: print("iter:", iteration)
``` 
before https://github.com/Nanne/pytorch-NetVlad/blob/master/main.py#L97 to get an indication of the progress.
